lib/shortcode.php

```diff
@@ -185,38 +182,40 @@ function anvato_shortcode($attr) {
 		$json['html5'] = true;
```

Comment:
these changes to json has to be also done for FIA.
If condition should be if ((function_exists('is_amp_endpoint') && is_amp_endpoint()) || defined('INSTANT_ARTICLES_SLUG') && is_feed( INSTANT_ARTICLES_SLUG))